nexus/utils/index.ts

```diff
@@ -1,6 +1,12 @@
 import { diffChars } from 'diff'
 import escapeHTML from 'escape-html'
 import { User, Thread } from '.prisma/client'
+import { PrismaClient } from '@prisma/client'
+
+export type Context = {
+  db: PrismaClient
+  request: { userId?: number }
+}
```

Comment:
In #394 we are adding an explicit `Context` type but you shouldn't need that here as mentioned above